client/components/article_list/index.coffee

```diff
@@ -19,9 +19,17 @@ module.exports = ArticleList = React.createClass
     else if result.scheduled_publish_at
       "Scheduled to publish " +
       "#{moment(result.scheduled_publish_at).fromNow()}"
+    else
+      "Last saved " +
+      "#{moment(result.updated_at).fromNow()}"
+
+  noResults: ->
+    if !@props.articles?.length
```

Comment:
Same here with !. Better to use `unless @props.articles?.length` Curious why you extracted this out to a function?